src/components/layout.css

```
@@ -102,3 +102,19 @@ h3 {
 @tailwind components;
 
 @tailwind utilities;
+
```

```suggestion

/* Ensure Footer always sticks to the bottom */
```
let's add a little comment here to explain what we are doing